src/Microsoft.AspNet.Authentication/DataHandler/TicketSerializer.cs

```diff
@@ -36,98 +39,177 @@ public virtual AuthenticationTicket Deserialize(byte[] data)
             }
         }
 
-        public static void Write([NotNull] BinaryWriter writer, [NotNull] AuthenticationTicket model)
+        public virtual void Write([NotNull] BinaryWriter writer, [NotNull] AuthenticationTicket ticket)
```

Comment:
Ah, I missed the Twitter cross-over. That's pretty sloppy, but it doesn't look easy to fix. Disregard.